### DISCUSSION TITLE
'Partition coefficient (water/container)' that cannot be found

### DISCUSSION THREAD

**Discussion by SCZ1174** [Open-Systems-Pharmacology/Forum#1559] [Questions & Problems] [answered]
Hello everyone,
I am experiencing an error reported while constructing a PBPK model for a pregnant population for metronidazole.
When I am adding a new simulation, I am prompted with an error
![DLXY2NQZUV)8TJ`PEP1X$VN](https://github.com/Open-Systems-Pharmacology/Forum/assets/86552551/af4d6f0c-21b6-495b-8a3d-8c65dd6c2e70)
![02B_DS}_D1O C1%R2~3 563](https://github.com/Open-Systems-Pharmacology/Forum/assets/86552551/05ac062e-d3b5-4f80-9e6e-4d73e918f22c)
Of note, I did not have this problem when I tried it in the previous version of the OSP suite and was able to add the simulation without any problems, but after updating the version, this problem appeared. I tried to repeat this in the previously saved success file as well as in the tutorial file, but the error also appeared.

**Comment by Yuri05** [comment]
Please check these discussions:
https://github.com/Open-Systems-Pharmacology/Forum/discussions/949
https://github.com/Open-Systems-Pharmacology/Forum/issues/46
https://github.com/Open-Systems-Pharmacology/Forum/discussions/792

**Reply by SCZ1174** [reply, marked as answer]
Thank you for your reply, I have solved the problem. It was an error that occurred when I was executing the simulation and selecting the module object.
